pkg/extender/extender.go

```diff
@@ -63,11 +88,17 @@ func (e *Extender) Start() error {
 	e.server = &http.Server{Addr: ":8099"}
 
 	http.HandleFunc("/", e.serveHTTP)
+	http.Handle("/metrics", promhttp.Handler())
```

Comment:
I think this needs to be registered at a higher level, since the metrics will be used by other modules too. If the extender is disabled it'll end up not registering the metrics handler

Reply:
should we have seperate prom metric server, which can be disable at start?